lego/apps/polls/tests/test_models.py

```diff
@@ -0,0 +1,15 @@
+from lego.apps.polls.models import Poll
+from lego.apps.users.models import User
+from lego.utils.test_utils import BaseTestCase
+
+
+class PollMethodTest(BaseTestCase):
+    fixtures = ["test_users.yaml", "test_polls.yaml"]
+
+    def setUp(self):
+        self.user = User.objects.get(username="test1")
+        self.poll = Poll.objects.get(pk=1)
+
+    def test_str(self):
+        title = "Test poll 1"
+        self.assertEqual(str(self.poll), title)
```

Comment:
Would be nice with some logic testing.

Eg. voting increases the count of total votes, and the count of users that has voted. And wise versa for the opposite.